### PR TITLE
sanitize some cluster up imports

### PR DESCRIPTION
**Pull request by mfojtik**
* first commit move modify SCC to external (@enj do we use this anywhere else? the only usage I found was setup persistent volume component in cluster up...)
* second commit moves router to use external client/types
* last commit removes post-startup mutations which were there to support `oc cluster join` (which we don't support) and moves webconsole oauth client to external

This is moving us closer to cluster up extraction. 
/cc @deads2k

### CODE REVIEW

File: pkg/oc/cli/admin/policy/authz_helpers.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	if name == "~" {
+		return []string{`may not equal "~"`}
+	}
+	return nil
+}
+
+func determineUserKind(user string) string {
+	kind := "User"
+	if len(validateUserName(user, false)) != 0 {
+		kind = "SystemUser"
```
Comment (deads2k) [inline pkg/oc/cli/admin/policy/authz_helpers.go]: This distinction no longer exists.

File: pkg/oc/cli/admin/policy/authz_helpers.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	}
+	return kind
+}
+
+func determineGroupKind(group string) string {
+	kind := "Group"
+	if len(validateGroupName(group, false)) != 0 {
+		kind = "SystemGroup"
```
Comment (deads2k) [inline pkg/oc/cli/admin/policy/authz_helpers.go]: this distinction no longer exists

File: pkg/oc/cli/admin/policy/authz_helpers.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+		kind := determineGroupKind(group)
+		subjects = append(subjects, corev1.ObjectReference{Kind: kind, Name: group})
+	}
+
+	return subjects
+}
+
+// duplicated from the user/validation package.  We need to avoid api dependencies on validation from our types.
+// These validators are stable and realistically can't change.
+func validateUserName(name string, _ bool) []string {
```
Comment (deads2k) [inline pkg/oc/cli/admin/policy/authz_helpers.go]: which means this is dead.

File: pkg/oc/cli/admin/policy/authz_helpers.go (hunk 1 above)
Comment (mfojtik) [reply pkg/oc/cli/admin/policy/authz_helpers.go]: will remove

**Comment by mfojtik** [comment]
@deads2k removed dead stuff, ptal

File: pkg/oc/clusterup/docker/openshift/helper.go [hunk 4]
```diff
@@ -175,46 +173,6 @@ func (h *Helper) OtherIPs(excludeIP string) ([]string, error) {
 	return resultIPs, nil
 }
 
-// CheckNodes determines if there is more than one node that corresponds to the
-// current machine and removes the one that doesn't match the default node name
-func (h *Helper) CheckNodes(kclient kclientset.Interface) error {
```
Comment (deads2k) [inline pkg/oc/clusterup/docker/openshift/helper.go]: you sure this isn't needed to handle restart cases?

**Comment by deads2k** [comment]
one question.  lgtm otherwise.  I'll leave tagging to you.

**Comment by mfojtik** [comment]
fixed the restart case, adding lgtm per slack

**Pull request merged**